ccxt.d.ts

```diff
@@ -212,9 +215,12 @@ declare module 'ccxt' {
         tag?: string;
     }
 
-    // timestamp, open, high, low, close, volume
+    /** [ timestamp, open, high, low, close, volume ] */
     export type OHLCV = [number, number, number, number, number, number];
 
+    /** Request parameters */
+    type params = Dictionary<string | number>;
```

Comment:
Suggest making this uppercase so it's clear it's a type. It's odd reading it below in the function args as lowercase.

Reply:
Done.